js/src/momoa.d.ts

```diff
@@ -0,0 +1,113 @@
+
+//-----------------------------------------------------------------------------
+// Options
+//-----------------------------------------------------------------------------
+
+type MomoaMode = "json" | "jsonc";
```

Comment:
I would personally drop the Momoa- prefixes on all these types, since you have to import them from './momoa' anyway.

However, if you need to work with Node/Token/Options types from other packages, it would be a good idea to keep them.

Reply:
Initially I was getting collisions with `Range` in `dom.d.ts` until I figured out how to get rid of that. So yes, don't need the prefixes anymore.